jwst/datamodels/tests/test_schema_editor.py

```diff
@@ -0,0 +1,43 @@
+"""Tests for the schema editor
+
+Notes
+-----
+The tests do not have coverage. The original code was written without tests.
+The tests here are for later modifiations.
+"""
+from os import path as ospath
+import pytest
+
+from ..schema_editor import Schema_editor
+
+
+def t_path(partial_path):
+    """Construction the full path for test files"""
+    test_dir = ospath.dirname(__file__)
+    return ospath.join(test_dir, partial_path)
+
+
+@pytest.fixture
+def keyword_db():
+    """Define the keyword database"""
+    keyword_db = t_path(ospath.join('data', 'jwstkd'))
+    return keyword_db
+
+
+def test_just_run(_jail, keyword_db):
+    """Just run the editor"""
+
+    editor = Schema_editor(
+        input=keyword_db,
+        add=True, delete=True, edit=True, rename=True, list=True
```

Comment:
If the schema editor prompts for input while running, how does the test work?

Reply:
When run from the API, the editor does not run in interactive mode without being passed an `Options` object; see the `scripts/schema_editor` file for details. So, this is basically doing a run to just list what the changes are without actually doing any of them. Simply a canary test.